src/utils/actions.js

```diff
@@ -0,0 +1,6 @@
+export const ACTIONS = {
```

Comment:
you may want this to be inside `Timer.js`, seems a bit out of place the actions of a component reducer being here